src/utils/transaction.ts

```diff
@@ -38,6 +39,108 @@ import { CROSS_CHAIN_ACTION_STATUS } from '../constants/transactionDispatcherCon
 import { ITransactionBlock } from '../types/transactionBlock';
 import { POLYGON_USDC_CONTRACT_ADDRESS } from '../constants/assetConstants';
 
+const fetchBestRoute = async (
+  sdk: EtherspotSdk,
+  fromChainId: number,
+  toChainId: number,
+  fromAmount: BigNumber,
+  fromTokenAddress: string,
+  toTokenAddress: string,
+  toAddress: string
+) => {
+  try {
+    const routes = await sdk.getAdvanceRoutesLiFi({
+      fromChainId,
+      toChainId, //Polygon
+      fromAmount,
+      fromTokenAddress,
+      toTokenAddress,
+      toAddress,
+    });
+    const bestRoute = routes.items.reduce((best: any, route) => {
+      if (!best || BigNumber.from(best['toAmount']).lt(route.toAmount)) return route;
```

Comment:
Also this isn't wrong, just a consistency thing - and also added a check to see if `toAmount` exists on `best`, since it's of an `any` type
```suggestion
      if (!best?.toAmount || BigNumber.from(best.toAmount).lt(route.toAmount)) return route;
```